orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/PipelineExpressionEvaluator.java

```diff
@@ -148,6 +148,12 @@ public PipelineExpressionEvaluator(
             pluginManager,
             expressionProperties);
     initExecutionAwareFunctions(expressionFunctionProviders);
+    parser =
+        new SpelExpressionParser(
+            expressionProperties.getMaxExpressionLength() > 0
+                ? new SpelParserConfiguration(
```

Comment:
too bad there's no builder for SpelParserConfiguration.

Reply:
This code can be dropped moving beyond springframework >=6.1.3.